buildbot/configure.py

```diff
@@ -43,6 +44,13 @@ def do_configure(args):
         libclc_targets_to_build = 'nvptx64--;nvptx64--nvidiacl'
         sycl_build_pi_cuda = 'ON'
 
+    if args.rocm:
+        llvm_targets_to_build += ';AMDGPU'
+        # TODO 
```

Comment:
Could you elaborate more on what exactly should be done here?
My guess is that when we build DPC++ with both CUDA and ROCM support, libclc should be added once. Right?

Reply:
Well, it's the same as my thoughts.

Reply:
At least for plugin we can use hip<->cuda [source code compatibility](https://github.com/ROCm-Developer-Tools/HIP/blob/main/docs/markdown/hip_faq.md#can-a-hip-binary-run-on-both-amd-and-nvidia-platforms)

Reply:
Could you add this clarification to the comment, please?